### DISCUSSION TITLE
[네트워크] 반이중화 통신과 전이중화 통신

### DISCUSSION THREAD

**Discussion by hyelim-choi** [GDSC-SWU/2023-CS-Study#3] [Q&A]
반이중화 통신과 전이중화 통신을 비교하여 설명해주세요

**Comment by yerim123456** [comment]
- 전이중화 통신은 유선 LAN에서 사용하는 통신으로, 양쪽 장치가 동시에 송수신이 가능합니다.
- 반이중화 통신은 무선 LAN에서 사용하는 통신으로, 양쪽 장치의 통신은 가능하나, 한 번에 한 방향만 통신이 가능합니다.
-> 그렇기 때문에 반이중화 통신에서는 충돌이 발생할 수 있는데, 이를 방지하기 위한 시스템이 필요합니다.  
-> 예를 들어 CSMA/CD 방식은 캐리어 감지 등으로 사전에 가능한 충돌을 방지합니다. 
-> CSMA/CD의 충돌 방지 단계는 4가지로 나뉩니다.
     1. 데이터를 송신하기 전 무선 매체를 살핌
     2. 캐리어 감지를 통해 회선이 비어있는지 확인
     3. IFS라고 랜덤 값을 기반으로 정해진 시간만큼 기다렸다가 사용 중이면 그 간격을 넓혀 체크
     4. 회선이 비어있다는 판단이 나오면 데이터 송신

** 저는 이렇게 알고 있는데 혹시 아니거나, 덧붙이고 싶으신 부분이 있으시다면 추가로 답변 남겨주시면 감사할 것 같습니다:)
